tests/modules/playarte.crawler.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+let expect = require('chai').expect;
+let path   = require('path');
+
+let PlayarteCrawler = require(path.join(__dirname, '../../modules/', 'cinemark.crawler.class'));
+
+
+describe('PlayarteCrawler', () => {
+    let Crawler;
+
+    beforeEach(function(){
+        Crawler = new PlayarteCrawler();
+    })
+
+    it('getPage(): Should return a page with title containing \'Playarte\'', (done) => {
+        let url = 'http://www.playartecinemas.com.br/';
+        Crawler.getPage(url)
+            .then(function($){
+                expect($("title").html().toLowerCase())
```

Comment:
Remember that the crawlers classes should only return the JSON already treated.

The tests consists in check if values from schedule of cinema are not null.